airflow/kubernetes/pod_launcher.py

```diff
@@ -162,22 +166,95 @@ def base_container_is_running(self, pod: V1Pod):
             return False
         return status.state.running is not None
 
+    @tenacity.retry(
+        stop=tenacity.stop_after_attempt(3),
+        wait=tenacity.wait_exponential(),
+        reraise=True,
+    )
+    def _read_pod_log_chunk(self, pod: V1Pod, last_line: bytes) -> HTTPResponse:
+        # The CoreV1Api doesn't support since_time even though the API does, so we must use
```

Comment:
This is a limitation in the python module right?

Reply:
The reason I ask is parsing the log line manually is fragile and error, prone, espeically in the case of user-configurable logging config, and task output that may print multiple lines, meaning not every line will start with a timestamp even _if_ the log format is stock.

Reply:
It's a limitation of the python module, but that's because for some reason they didn't include since_time in the API spec.

The timestamp comes from the output of the kubernetes API and will always have a timestamp at the front no matter how many lines are printed or what the line content is

Reply:
Has anyone reported this to the upstream project? We should and put a link to that issue here so we can remove this hack when they eventually fix it.

Reply:
@dorranh referenced these issues with the upstream project, but they're all closed.

https://github.com/kubernetes/kubernetes/issues/44340
https://github.com/kubernetes/kubernetes/issues/59477
https://github.com/kubernetes/kubernetes/issues/28369

Reply:
I'll try to get the build passing this weekend if I get some time. I've been having a tough time getting all of the integration tests to pass in CI. When I try the ones that fail in CI on my local machine they pass, so having a tough time debugging them.